lib/God/Methods.js

```diff
@@ -242,7 +257,7 @@ module.exports = function(God) {
         if (mode.indexOf('cluster') === 0)
           treekill(parseInt(pid));
         else
-          process.kill(-(parseInt(pid)), 'SIGINT');
```

Comment:
Just for my knowledge, why the behaviors differ in terms of mode.indexOf('cluster') ? In other word, why couldn't we use treekill instead of `process.kill(-pid)` everywhere?